extensions/Vendor/FRAUNHOFER_materials_pbr/schema/materials_pbr_metal_roughness.schema.json

```diff
@@ -0,0 +1,33 @@
+{
+    "$schema" : "http://json-schema.org/draft-03/schema",
+    "title" : "material",
+    "type" : "object",
+    "description" : "An attribute or uniform input to a technique, and an optional semantic and value.",
+    "extends" : { "$ref" : "glTFProperty.schema.json" },
+    "properties" : {
+    	"baseColorFactor" : {
+    		"type" : { "$ref" : "arrayValues.schema.json" },
+    		"description" : "The material's base color factor.",
+            "default" : [1,1,1,1]
+    	},
+        "metallicFactor" : {
```

Comment:
Throughout, JSON schema can define min/max (exclusive and inclusive) values.  For example, see https://github.com/KhronosGroup/glTF/blob/master/specification/schema/camera.perspective.schema.json#L14-L27